documentation/src/main/asciidoc/topics/proc_configuring_server_keystores.adoc

```diff
@@ -65,13 +65,13 @@ include::yaml/server_ssl_identity_truststore.yaml[]
 
 == SSL/TLS Certificate rotation
 
-SSL/TLS certificates have an expiration date, after which they will no longer be valid.
-The process of renewing a certificate is also known as *rotation*.
+SSL/TLS certificates have an expiration date, after which they are no longer be valid.
```

Comment:
if you change the tense, also remove the "be"

Reply:
sorry, fixed